importer/api/__init__.py

```diff
@@ -43,7 +43,7 @@ def import_course_from_file(filename, repo_id, user_id):
     try:
         extract(path=filename, to_path=tempdir, method="safe")
     except ArchiveException as ex:
-        log.debug("failed to extract: %s", ex)
+        log.error("failed to extract: %s", ex)
```

Comment:
This seems out of scope for this PR, also, if you use log.execption it prints the stack trace and you don't have to pass the exception into it.